### PR TITLE
Build/postinstall compile

### PR DESCRIPTION
**Pull request by anoff**
If installed the package automatically creates `build/ModelRepository.abi` from source.

### CODE REVIEW

File: bin/extract-abi [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+ * Usage: extract-abi build/contracts/mycontract.json > contractabi.json
+ */
+const fs = require('fs')
+
+const filepath = process.argv[2]
+if (!filepath || !fs.statSync(filepath)) throw new Error(`File does not exist: ${filepath}`)
+
+const json = JSON.parse(fs.readFileSync(process.argv[2], 'utf8'))
+
+process.stdout.write(Buffer.from(JSON.stringify(json.abi)) + '\n')
```
Comment (axelhodler) [inline bin/extract-abi]: would `console.log(JSON.stringify(json.abi))` accomplish the same here?

Comment (anoff) [reply bin/extract-abi]: yes

Comment (anoff) [reply bin/extract-abi]: not 💯 sure how `console.log` and `process.stdout` behave different when piping though. `stdout` feels like the safer choice to work with pipes etc

Comment (axelhodler) [reply bin/extract-abi]: image

lets go for `console.log` then, seeing `\n` and `Buffer` and `stdout` makes me not feeling node-like ;)

**Pull request merged**